lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -301,6 +369,94 @@ class PatternStopContents extends Component<Props, State> {
             </FormGroup>
           </Col>
         </Row>
+        {/* Pickup and drop off type selectors */}
+        <Row>
+          <Col xs={6}>
+            <FormGroup
+              controlId='pickupType'
+              bsSize='small'>
+              <ControlLabel
+                className='small'
+                title='Define the pickup method/availability at this stop.'
+              >
+                Pickup type
```

Comment:
I'd be curious about what others may have to say. With the common options from `pickupDropoffOptions`, does it make sense for this label to just say "Pickup"?

Reply:
We can try "Pickup" and "Drop off" and we can see how it feels.